server/models/Event.ts

```diff
@@ -88,7 +88,7 @@ class Event extends IdModel {
 
   @ForeignKey(() => Collection)
   @Column(DataType.UUID)
-  collectionId: string;
+  collectionId: string | null;
 
   @BelongsTo(() => Team, "teamId")
   team: Team;
```

Comment:
I wonder why these fields weren't nullable already, since I see from the database schema they are?

Reply:
👍  Just a mistake

Reply:
alright, I will include that later.